gammagl/models/grade.py

```diff
@@ -0,0 +1,103 @@
+import tensorlayerx as tlx
+from gammagl.layers.conv import GCNConv
+import numpy as np
+
+
+# Multi-layer Graph Convolutional Networks
+class GCN(tlx.nn.Module):
+    def __init__(self,in_dim, out_dim,act_fn, num_layers = 2):
+        super(GCN, self).__init__()
+        assert num_layers>=2
+        self.num_layers=num_layers
+        self.convs=tlx.nn.ModuleList()
+
+        self.convs.append(GCNConv(in_dim,out_dim*2))
+        for _ in range(self.num_layers-2):
+            self.convs.append(GCNConv(out_dim*2,out_dim*2))
+
+        self.convs.append(GCNConv(out_dim*2,out_dim))
+        self.act_fn=act_fn
+
+
+    def forward(self, feat,edge_index):
+        for i in range(self.num_layers):
+            feat=self.act_fn(self.convs[i](feat,edge_index))
+        return feat
+
+# Multi-layer(2-layer) Perceptron
+class MLP(tlx.nn.Module):
```

Comment:
Ditto, we already have MLP module.

Reply:
The GRADE model has the same network structure as GRACE，so I try to write it as the  same as GRACE. If it needs to be modified, I can replace it with MLP module and the result will be the same.